package.json

```diff
@@ -1,30 +1,34 @@
 {
-    "name": "nsp-formatter-multi",
-    "version": "0.0.9",
-    "description": "Outputs a file for slack, reporting and outputs default NSP report",
-    "main": "index.js",
-    "engines": {
-        "node": ">=4"
-    },
-    "scripts": {
-        "nsp": "nsp check"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/awolden/nsp-formatter-multi.git"
-    },
-    "keywords": [
-        "nsp",
-        "formatter",
-        "minimal"
-    ],
-    "author": {
-        "name": "Alexander Wolden",
-        "email": "awolden@gmail.com"
-    },
-    "license": "Apache-2.0",
-    "dependencies": {
-        "nsp": "^2.6.1",
-        "text-table": "^0.2.0"
-    }
+  "name": "nsp-formatter-multi",
+  "version": "0.0.10",
```

Comment:
considering there are no major versions... I'm just incrementing the base version.  http://semver.org/#spec-item-4